packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -340,4 +381,78 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     this[$ensureMaterialIsLoaded]();
     return (this[$sourceObject] as DefaultedMaterial).alphaMode;
   }
+
+  /**
+   * PBR Next properties.
+   */
+  get emissiveStrength(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].emissiveIntensity;
+  }
+
+  get clearcoatFactor(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].clearcoat;
+  }
+
+  get clearcoatRoughnessFactor(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].clearcoatRoughness;
+  }
+
+  get clearcoatTexture(): TextureInfo {
+    this[$ensureMaterialIsLoaded]();
+    return this[$clearcoatTexture];
+  }
+
+  get clearcoatRoughnessTexture(): TextureInfo {
+    this[$ensureMaterialIsLoaded]();
+    return this[$clearcoatRoughnessTexture];
+  }
+
+  get clearcoatNormalTexture(): TextureInfo {
+    this[$ensureMaterialIsLoaded]();
+    return this[$clearcoatNormalTexture];
+  }
+
+  get clearcoatNormalScale(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].clearcoatNormalScale.x;
+  }
+
+  setEmissiveStrength(emissiveStrength: number) {
+    this[$ensureMaterialIsLoaded]();
+    for (const material of this[$correlatedObjects] as
+         Set<MeshPhysicalMaterial>) {
+      material.emissiveIntensity = emissiveStrength;
```

Comment:
LGTM, but I believe these are simplified compared to the existing setters right (recall our duplicated state)? We should change the others to match this style and check to ensure that doesn't break anything before we adopt it for these. Let's add that in before we get too deep into testing these new ones.